federated_learning/substra/assets/objective/metrics.py

```diff
@@ -5,7 +5,7 @@
 
 
 class MonaiMetrics(tools.Metrics):
-    dice_metric = DiceMetric(include_background=True, to_onehot_y=False, sigmoid=True, reduction="mean")
+    dice_metric = DiceMetric(include_background=True, reduction="mean")
```

Comment:
Maybe you should also add `post_trans = Compose([Activations(sigmoid=True), AsDiscrete(threshold_values=True)])
    # try to use all the available GPUs` in this FL example?